predictoor/pdr-parameters.md

```diff
@@ -0,0 +1,23 @@
+# Predictoor Parameters
+
+## Feeds Published
+For testnet, there are 10 feeds: X/USDT pair for each of the top-10 coins by market cap (ignoring stablecoins), 5m timescales, on Binance, >0% fees on Binance. Paid feeds. The coins are: X = BTC, ETH, BNB, XRP, ADA, DOGE, SOL, LTC, TRX, DOT
+
+For mainnet, tentatively, there will be 20 feeds:  
+- 10x at 5m timescales like above  
+- 10x at 60m timescales like above  
```

Comment:
Change this section to:

```
For both testnet and mainnet, there are 20 feeds:
- X/USDT pair for each of the top-10 coins by market cap (ignoring stablecoins), on Binance, 5m timescales, on Binance. The coins are: X = BTC, ETH, BNB, XRP, ADA, DOGE, SOL, LTC, TRX, DOT
- Like above, but for 60m timescales
```

Why: 
- the info is no longer tentative
- we have 5m & 60min on both networks now
- Binance info on ">0% fee" is no longer relevant, it's all like that
- FYI the blog post is also updated accordingly.

Reply:
Done